modules/elasticsearch-index/src/main/java/org/opencastproject/elasticsearch/index/objects/event/EventSearchQuery.java

```diff
@@ -359,7 +509,16 @@ public EventSearchQuery withSeriesId(String seriesId) {
    *
    * @return the series identifier
    */
-  public String getSeriesId() {
+  public String getSeriesIdValue() {
+    return seriesId.getValue();
+  }
+
+  /**
+   * Returns the series identifier of the recording.
+   *
+   * @return the series identifier
+   */
+  public EventSearchQueryField<String> getSeriesId() {
```

Comment:
This is why [CI fails](https://github.com/opencast/opencast/actions/runs/16772761044/job/47491650160?pr=6139). [TestSeriesEndpoint uses this](https://github.com/opencast/opencast/blob/2b1701969a5ae14dbaa77e94af88dbfa33568e66/modules/admin-ui/src/test/java/org/opencastproject/adminui/endpoint/TestSeriesEndpoint.java#L418-L471) but still expects it to be a `String`. You might want to check other usages of these methods that now return your wrapper. 🤔